doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -278,125 +281,125 @@ Boot disks          RAID 1 RAID 1
 -  In case of DPDK usage:
 
 +-------------------------------------------------------------+--------+
-| Layer                                                       | Descr  |
```

Comment:
I think a bit wider 2nd column would be needed here also.

Reply:
@CsatariGergely there are too many tables, it would be impossible to modify the column sizes.

Reply:
@CsatariGergely can you review again?